src/__tests__/Storage.ts

```diff
@@ -0,0 +1,16 @@
+import MockStorage from '../__mocks__/MockStorage';
+import Storage from '../Storage';
+
+describe('Storage', () => {
+  const storage = new Storage({
+    storage: new MockStorage(),
+  });
+
+  it('writes data to persistent storage', async () => {
+    await expect(storage.write('yo yo yo')).resolves.toBe(undefined);
+  });
+
+  it('reads data from persistent storage', async () => {
+    await expect(storage.read()).resolves.toBe('yo yo yo');
```

Comment:
It's best not to assume these two tests will run in order. (It's not guaranteed.)

I suggest doing an explicit `write` before the `read` in the second test — or you can combine them.

Reply:
combined them & also added one for `storage.purge()` 😀